DDD Workshop.ipynb

```diff
@@ -0,0 +1,547 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Data-Driven Decisions\n",
+    "\n",
+    "In this piece, we'll get down to the core of what data means, how we can extract knowledge from information from data, and how to use it as a savvy businessperson, researcher, or analyst. You won't need any preparation, but there will be some code available for the aspiring programmers to use and modify. You will walk away from this page with a better understanding of what your data needs or needs not and how to most effectively to use it.\n",
+    "\n",
+    "\n",
+    "\n",
+    "# Part 1: Information vs Data\n",
+    "\n",
+    "Webster's online dictionary defines Data as factual measurements or statistics used as a basis for actions such as reasoning, deliberations, or calculation. We want to pay specific attention to the word \"basis\" here, as it highlights the fact that data can be, quite easily, manipulated. If we don't want to be susceptible to this manipulation, then we should make sure we turn that data into information. We see that Information's entry is \"knowledge obtained from investigation, study, or instruction,\" and that is really what's at play when we're making smart, data-driven decisions:\n",
```

Comment:
- Instead of Data, use "data" 
- Remove the , between "here" and "as" 
- Would remove the commas around "quite easily" to emphasize that it's easy to manipulate data
- Is "Information's entry" referring to the definition in the Webster's online dictionary? Could clarify if so; maybe something like this: We also see that "information" is the same diction is knowledge obtained from..."